### PR TITLE
Please check out my English HW3

### PR DESCRIPTION
**Pull request by VladimirPetranovski**

### CODE REVIEW

**Review by olgaz-skiba — APPROVED** [review]
Lgfm

File: homework/english/petranovski_vladimir/lecture_3/grammar.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+11. Я пойду дополнительно изучать англйиский на следующей неделе, потому что это необходимо для моей работы.
+    I'll go to study English additionally next week because it's necessary for my work.  
+12. После сдачи проекта он даст мне доступ к этим папкам.
+    After passing the project he'll give me a access by these folders.
+13. Наша команда скоро начинает работать над новым проектом.
+    Our team will start on a new project soon.
+
+### Упражнение 2. Образуйте вопрос к предложению
+
+1. Nina will be at school in 10 minutes. Where will Nina be at school?
```
Comment (Alex99905) [inline homework/english/petranovski_vladimir/lecture_3/grammar.md:34]: Where will Nina be in 10 minutes.

File: homework/english/petranovski_vladimir/lecture_3/grammar.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+2. The competition will start tomorrow. When will the competition start?
+3. Ben will buy a new bicycle. What will Bill buy?
+4. Sam will come soon. Who soon will Sam come?
```
Comment (Alex99905) [inline homework/english/petranovski_vladimir/lecture_3/grammar.md:37]: Who will come soon?

File: homework/english/petranovski_vladimir/lecture_3/grammar.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    Our team will start on a new project soon.
+
+### Упражнение 2. Образуйте вопрос к предложению
+
+1. Nina will be at school in 10 minutes. Where will Nina be at school?
+2. The competition will start tomorrow. When will the competition start?
+3. Ben will buy a new bicycle. What will Bill buy?
+4. Sam will come soon. Who soon will Sam come? 
+5. We will go to the cinema at 6 o’clock. What time will we go to cinema?
+6. They will fly to Australia next week. Where will they fly to Australia?
```
Comment (Alex99905) [inline homework/english/petranovski_vladimir/lecture_3/grammar.md:39]: Where will they fly next week?

**Review by Alex99905 — APPROVED** [review]

**Review by youRaFyodorov — APPROVED** [review]

**Pull request merged**